docs/system/wind_turbine.md

```diff
@@ -1,7 +1,147 @@
+            <fig-caption src="system/savonius_turbine.svg" caption="Two-scooped savonius turbine [1]" style="width:25%"  num="1" />
+        </a>
+    </center>
+</figure>
+
+The mathematical equation to describe the force on the scoops is given by
+
+$$F = \frac{c_w}{2} \cdot \varrho_{air} \cdot A \cdot v_{air}^2$$
+
+where $A$ is the cross-sectional area of the scoop, $v_{air}$ is the windspeed, $\varrho_{air}$ is air density and $c_w$ is the drag coeeficient specific for the geometry used. The coefficient is derived via measurements and can be found in corresponding tables. The force driving the rotation is the difference between the coefficients:
+
+$$\Delta F = \frac{c_{w,1}}{2} \cdot \varrho_{air} \cdot A \cdot v_{air}^2 - \frac{c_{w,2}}{2} \cdot \varrho_{air} \cdot A \cdot v_{air}^2 = \frac{c_{w,1} -c_{w,2}}{2} \cdot \varrho_{air} \cdot A \cdot v_{air}^2$$
+
+
+A Savonius rotors rotation speed can not exceed the wind velocity, so the Tip Speed Ratio (TSR) defined as
+
+$$\lambda = \frac{\omega r}{v_{air}}$$
+
+with $\omega$ as angular frequency and $r$ as radius, is always $\lambda\le1$. Due to their low power coeefficient ($c_p \lt 25\%)$ and low rotatioal speed, Savonius turbines are not commonly used for power generation. There are however cases were a high starting torque, simple design and low maintenance requirements are beneficial.
+
+### Lift Runner
+
+Lift running turbines use airfoils as blade geometry and are mostly divided into VAWT and Horizontal Axis Wind Turbines (HAWT). There are other hybrids and experimental types of turbines, but those are not commonly used. Most lift types have a much higher TSR ($\lambda$=2...12) [4] than drag types thus the wind velocity at the blade is much higher, resulting in better power coeeficients. All lift types have a relativly low starting torque, resulting in a high cut-in windspeed.
+
+<figure>
+    <center>
+        <a href="https://commons.wikimedia.org/wiki/File:Airfoil_lift_and_drag.svg#/media/File:Airfoil_lift_and_drag.svg">
+            <fig-caption src="system/airfoil_lift_and_drag.svg" caption="Basic principle of lift on an airfoil [2]" style="width:50%"  num="2" />
+        </a>
+    </center>
+</figure>
+
+#### HAWT
+
+Horizontal turbines are most commonly seen in commercial wind energy production due to there high power efficiency ($c_p \approx50\%$) and scalability. To control rotational speed, large systems use active pitch controller where each blade can be individually pitched to adapt generated torque. Small turbines with fixed blades need either a passive stall design or a yaw mechanism to effectivly lower the wind-facing area of the turbine. Most systems will have mechanical or hydraulic brake systems for storm protection. Since the relative windspeed attacking the blade changes along the radius, the angle of attack also changes which leads different optimal blade geometries along the blade [4]. 
```

Comment:
```suggestion
Horizontal turbines are most commonly seen in commercial wind energy production due to their high power efficiency ($c_p \approx50\%$) and scalability. To control rotational speed, large systems use active pitch controllers where each blade can be individually pitched to adapt the generated torque. Small turbines with fixed blades need either a passive stall design or a yaw mechanism to effectively lower the wind-facing area of the turbine. Most systems will have mechanical or hydraulic brake systems for storm protection. Since the relative wind speed attacking the blade changes along the radius, the angle of attack also changes, which results in different optimal blade geometries along the blade [4]. 
```